.travis.yml

```diff
@@ -104,6 +104,7 @@ install:
         python -c 'import mne; mne.datasets.testing.data_path(verbose=True)';
         if [ "${DEPS}" == "full" ]; then
           export FREESURFER_HOME=$(python -c 'import mne; print(mne.datasets.testing.data_path())');
+          export MNE_SKIP_FLASH_CALL=1;
```

Comment:
Ahh, you replied below:

> Yeah, this is to avoid the freesurfer calls.

Can you add a comment somewhere saying that it's done only to avoid a command that isn't available on CIs?